src/style.css

```diff
@@ -1 +1,24 @@
 /* add styles here */
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  color: #f3f3f3;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-size: 16px 16px;
+  margin-right: 6px;
+  margin-top: 6px;
+}
+
+.active {
+  background-image: url(images/star-active.svg);
```

Comment:
Ensure that the 'images/star-active.svg' file exists in the specified path. If the image is missing, the active stars will not display correctly.